charts/app-stable-diffusion/templates/_helpers.tpl

```diff
@@ -51,6 +51,28 @@ app.kubernetes.io/name: {{ include "app.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
 
+{{/*
+Common labels
+*/}}
+{{- define "stablestudio.labels" -}}
+application: stable-diffusion-ss
```

Comment:
We'll probably have a dedicated preset for stable studio (CPU-only one).

Also, we decided to use label `platform.apolo.us/preset: {{ .Values.preset_name }}` to track resource usage within apps, could you please add a dedicated label for ss too? We will pass it's value from within mlops-app-deployment job